addons/rook-ceph/disable

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+import subprocess
+
+import click
+
+SNAP = pathlib.Path(os.getenv("SNAP") or "/snap/microk8s/current")
```

Comment:
When is the second part (after the "or") called?

Reply:
In cases where the SNAP var can be an empty string. Just as a failsafe.